charts/cluster-scanner/values.yaml

```
@@ -156,15 +156,19 @@ imageSbomExtractor:
   pprofEnabled: false
 
   cache:
-    # The type of cache to use, either `local` or `redis`. When setting `redis`, both Redis and the local cache will be used.
+    # The type of cache to use.
+    # Allowed values are `local`, `distributed` and `distributed,local`.
+    # When specified more than one, the cache precedence will be applied from the left to the right.
```

I think the precedence is applied from right to left, since you first check whether the cache on the right does have the data.

```suggestion
    # When specified more than one, the cache precedence will be applied from right to left.
```